Dockerfile

```diff
@@ -1,8 +1,48 @@
-FROM clouder/base:3.4
-MAINTAINER Ted Salmon tsalmon@laslabs.com
+FROM alpine:latest
+MAINTAINER LasLabs Inc <support@laslabs.com>
 
-RUN apk add --no-cache samba-dc
 
-# Remove default Samba configs
-RUN rm -f /etc/samba/smb.conf
-RUN rm -rf /var/lib/samba/*
+ENV SAMBA_DC_ETC=/etc/samba
+ENV SAMBA_DC_DATA=/var/lib/samba
+ENV SAMBA_DC_LOGS=/var/log/samba
+ENV SAMBA_DC_REALM='corp.example.net'
+ENV SAMBA_DC_DOMAIN='EXAMPLE'
+ENV SAMBA_DC_ADMIN_PASSWD='5u3r53cur3!'
+ENV SAMBA_DC_DNS_BACKEND=SAMBA_INTERNAL
+
+# Install
+RUN apk add --no-cache samba-dc supervisor \
+    # Remove default config data, if any
+    && rm -f /etc/samba/smb.conf \
+    && rm -rf /var/lib/samba/* \
+    # Configure
+    && samba-tool domain provision --domain=$SAMBA_DC_DOMAIN \
```

Comment:
Lint seems broken, but it would be failing here. You should always quote encapsulate vars
```
--domain="${SAMBA_DC_DOMAIN}"
```